airbyte-ci/connectors/metadata_service/lib/metadata_service/validators/metadata_validator.py

```diff
@@ -61,30 +61,36 @@ def validate_all_tags_are_keyvalue_pairs(metadata_definition: ConnectorMetadataD
     return True, None
 
 
-def pre_upload_validations(metadata_definition: ConnectorMetadataDefinitionV0) -> ValidationResult:
-    """
-    Runs all validations that should be run before uploading a connector to the registry.
-    """
-    validations = [
-        validate_all_tags_are_keyvalue_pairs,
-        validate_at_least_one_langauge_tag,
-    ]
+PRE_UPLOAD_VALIDATORS = [
+    validate_all_tags_are_keyvalue_pairs,
+    validate_at_least_one_language_tag,
+]
 
-    for validation in validations:
-        valid, error = validation(metadata_definition)
-        if not valid:
-            return False, error
+POST_UPLOAD_VALIDATORS = PRE_UPLOAD_VALIDATORS + [
+    validate_metadata_images_in_dockerhub,
+]
 
-    return True, None
 
+def validate_and_load(file_path: pathlib.Path, extra_validators_to_run: Optional[List[Callable]] = None) -> Tuple[Optional[ConnectorMetadataDefinitionV0], Optional[ValidationError]]:
```

Comment:
nit: I think we always want to validate in someway now. So perhaps `validators_to_run` is not a Optional argument